src/components/form-intro.js

```diff
@@ -0,0 +1,83 @@
+import {nanoid} from 'nanoid';
+import {useState} from 'react';
+import {useForm} from 'react-hook-form';
+
+console.clear();
+
+export default function IntroForm() {
+	const {
+		register,
+		handleSubmit,
+		formState: {errors},
+	} = useForm();
+
+	const [people, setPeople] = useState([
```

Comment:
Why did you name the state `people`? 🤔 As far as I understand this state will save restaurants, so it would be best to name it that way as well.